tutorials/advanced_features/Serialization.ipynb

```diff
@@ -1,182 +1,162 @@
 {
```

Comment:
<p>I think something is missing in the last sentence "as we'll <strong>see" </strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Good catch!</p>